src/index.html

```diff
@@ -21,7 +24,396 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="navi-bar">
+        <ul class="navi-menu">
+          <li class="navi-item">
+            <a
+              href="#"
+              class="is-active"
```

Comment:
You have used the class 'is-active' here, but it does not seem to be defined in any of your CSS/SCSS files. Make sure to define it, or remove it if it's not needed.